PeteUgwu/To-Do-list
# Testing To Do list: Add and Remove functions

## Testing To Do list: Add and Remove functions

- Use jest framework for testing.
- Mock a storage object to "imitate" localStorage operation.
- Make sure you group your tests using the describe() method.
- Mock HTML to test if add/delete functions add or remove exactly an element to/from the list in the DOM.
- fix linters.
